src/models/decomposition_models.jl

```diff
@@ -43,8 +43,8 @@ function MMI.fit(model::PCA, verbosity::Int, X)
     )
     cache = nothing
     report = (
-        indim=MS.size(fitresult,1),
-        outdim=MS.size(fitresult,2),
+        indim=MS.size(fitresult)[1]
```

Comment:
I checked this and could not reproduce an error:

```julia
using MLJBase; using MLJMultivariateStatsInterface

X, _ = make_regression();

model = PCA()
mach = machine(model, X) |> fit!

julia> report(mach)
(indim = 2,
 outdim = 2,
 tprincipalvar = 1.9889224728839578,
 tresidualvar = 0.0,
 tvar = 1.9889224728839578,
 mean = [-0.11837285204697945, 0.06638205203530174],
 principalvars = [1.1156200691372054, 0.8733024037467526],)
```

Could you say more about this proposed change (if really needed, should probably go in a separate PR I am happy to expedite).

Reply:
this was actually my bad on some version issues I think!